cmd/release-controller/http.go

```diff
@@ -562,6 +564,51 @@ func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (c *Controller) httpLatestAccepted(w http.ResponseWriter, req *http.Request) {
+	start := time.Now()
+	defer func() { glog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()
+
+	vars := mux.Vars(req)
+	streamName := vars["release"]
+
+	stream, err := c.imageStreamLister.ImageStreams(c.releaseNamespace).Get(streamName)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	r, ok, err := c.releaseDefinition(stream)
+	if err != nil || !ok {
+		if err == nil {
+			err = fmt.Errorf("did not find release config for '%s'", streamName)
+		}
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	tags := tagsForRelease(r)
+	if len(tags) == 0 {
+		err = fmt.Errorf("no tags found for stream '%s'", streamName)
+
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	latestTag := tags[0]
```

Comment:
Is this accepted?  I think you need to filter by accepted (`tagReferencesByPhase(Accepted)`)